mesheryctl/internal/cli/root/design/delete.go

```diff
@@ -64,9 +64,9 @@ mesheryctl pattern delete [file | URL]
 			err := utils.DeleteConfiguration(mctlCfg.GetBaseMesheryURL(), pattern, "pattern")
 			if err != nil {
 				utils.Log.Error(err)
-				return errors.Wrap(err, utils.PatternError(fmt.Sprintf("failed to delete pattern %s", args[0])))
+				return errors.Wrap(err, utils.DesignError(fmt.Sprintf("failed to delete design %s", args[0])))
```

Comment:
If you log here and return the error the error will be logged twice. Please remove one for all the instances.//
@alphaX86

Reply:
Done